src/tests/ui/planner/workitem-quickpreview.ts

```diff
@@ -284,4 +280,9 @@ export class WorkItemQuickPreview extends ui.BaseElement {
   async hasTitleError(error: string) {
     return await this.titleErrorMessage.getTextWhenReady() === error;
   }
+
+  async changeStateToClose() {
```

Comment:
Can you please change the method to a more generic `changeStateTo` ?
We can then use it as `changeStateTo(workitemStates.close)`, `changeStateTo(workitemStates.open)`, ..

Reply:
Yes, I can make this more generic.

Reply:
Done https://github.com/jarifibrahim/fabric8-planner/pull/6/commits/a89e0cd9fc037e07fcdefe5f0bb5a5db00a4bf11